fdbserver/masterserver.actor.cpp

```diff
@@ -1242,6 +1245,12 @@ ACTOR Future<Void> serveLiveCommittedVersion(Reference<MasterData> self) {
 			when(ReportRawCommittedVersionRequest req =
 			         waitNext(self->myInterface.reportLiveCommittedVersion.getFuture())) {
 				self->minKnownCommittedVersion = std::max(self->minKnownCommittedVersion, req.minKnownCommittedVersion);
+				if (SERVER_KNOBS->ENABLE_VERSION_VECTOR && req.tagSet.present()) {
+					if (req.version > self->ssVersionVector.maxVersion) {
```

Comment:
Would you please add a comment that this if-condition is not needed after wait-for-prev is ported to this branch? Thanks!